app/views/unattended/report_templates/subscription_-_entitlement_report.erb

```diff
@@ -22,35 +22,39 @@ require:
 <%-   search: should_filter && "pools_expiring_in_days = #{days_from_now}" -%>
 <%- ).each_record do |host| -%>
 <%-   host.pools.expiring_in_days(days_from_now).sort_by { |p| p.end_date }.each do |pool| -%>
-<%-     report_row(
-          'Host Name': host.name,
-          'Organization': host.organization,
-          'Lifecycle Environment': host.single_lifecycle_environment.name,
-          'Content View': host.single_content_view.name,
-          'Host Collections': host_collections_names(host),
-          'Virtual': host.virtual,
-          'Guest of Host': host.hypervisor_host,
-          'OS': host.operatingsystem,
-          'Arch': host.architecture,
-          'Sockets': host.sockets,
-          'RAM': host.ram,
-          'Cores': host.cores,
-          'SLA': host_sla(host),
-          'Products': host_products_names(host),
-          'Last Checkin': last_checkin(host).to_s,
-          'Subscription Name': sub_name(pool),
-          'Subscription Type': pool.type,
-          'Subscription Total Quantity': pool.quantity,
-          'Subscription Total Consumed': pool.consumed,
-          'Subscriptions Consumed by Host': host.filtered_entitlement_quantity_consumed(pool),
-          'Subscription SKU': sub_sku(pool),
-          'Subscription Contract': pool.contract_number,
-          'Subscription Account': pool.account_number,
-          'Subscription Start': pool.start_date,
-          'Subscription End': pool.end_date,
-          'Subscription Guest': registered_through(host),
-          'Days Remaining': pool.days_until_expiration
-          ) -%>
+<%-     unless host.organization.simple_content_access? -%>
+<%-       report_row(
+            'Host Name': host.name,
+            'Organization': host.organization,
+            'Lifecycle Environment': host.single_lifecycle_environment.name,
+            'Content View': host.single_content_view.name,
+            'Host Collections': host_collections_names(host),
+            'Virtual': host.virtual,
+            'Guest of Host': host.hypervisor_host,
+            'OS': host.operatingsystem,
+            'Arch': host.architecture,
+            'Sockets': host.sockets,
+            'RAM': host.ram,
+            'Cores': host.cores,
+            'SLA': host_sla(host),
+            'Role': host.purpose_role_status_label,
+            'Usage': host.purpose_usage_status_label,
```

Comment:
in light of my comments above, these should really be called "Role Status" and "Usage Status"

Reply:
Well if I change them to be purpose role and purpose usage, I think Role and Usage make sense again. I'll just remove the role and usage statuses.

Reply:
Sorry, I meant

- leave the statuses here, in the old Entitlement report, and call the columns 'Role Status' and 'Usage Status'; and
- in the new Host Installed Products report, call the columns 'Role' and 'Usage' and have the actual values.

Reply:
We could also have _both_ (status and value) columns in the Entitlement Report if you want.